### ISSUE TITLE
Calling of original methods when doubling

### ISSUE THREAD

**Issue by Beliaar** [bitwes/Gut#98]
I wanted to use doubling to test if a method calls another under certain circumstances. Currently double overrides all methods which makes this impossible to do.

A possible solution would be if one were able to pass a list of method names not to override so they retain their behaviour.

**Comment by bitwes** [comment]
Just putting this here for later.  Possible implementation syntax: 
```
stub(my_obj, 'some_method').to_call_super()
stub(my_obj, 'some_method').to_call_super().when_passed(...)
```

**Comment by bitwes** [comment]
I've implemented this in the next release branch currently named `v6.7.1`.  I've tested it, but I haven't dog-fooded it yet.  Any testing/feedback would be greatly appreciated.

I haven't done the documentation for it yet, but it works the way you would expect.  More info can be found in the CHANGES.MD for the branch:  https://github.com/bitwes/Gut/blob/v6.7.1/CHANGES.md

**Comment by bitwes** [comment]
The new `to_call_super` stub chain has been implemented in 6.8.0.  Check out the stubbing wiki page for more information.
